### ISSUE TITLE
Error: SendCoTAbstractController.py:23

### ISSUE THREAD

**Issue by w111a** [FreeTAKTeam/FreeTakServer#160]
I have found the following errors in my log. 

```
ERROR : 2021-05-28 14:10:53,235 : SendCoTAbstractController.py:23 : an undocumented CoT has been sent returning the error targetMunitions does not have specified attribute please post this as an issue on the FTS git so we can document and fix it
```

```
ERROR : 2021-05-28 14:11:08,327 : ClientReceptionHandler.py:72 : Exception other than broken pipe in monitor for data function [Errno 11] Resource temporarily unavailable
```

OS: Unraid 6.9.2
ATAK: 4.3.1.8
Device : SM-G781W
Android build:30
Docker build: skadakar
/
FreeTAKServer-Docker: 1.8.1

I have been trying to get an ssl connection working and manually imported both certs that the server made.  TCP works well but ssl disconnects after 20-30 seconds. 
Thanks

**Comment by brothercorvo** [comment]
@naman108 I think we have fixed it, moving from error to warning?

**Comment by brothercorvo** [comment]
this should be fixed in 1.9. closing for now
